src/common/util/firebase.ts

```diff
@@ -229,7 +230,12 @@ export function indexParsedSentence(sentence: string, tokens: string[], js: stri
 export async function getRecentGamesByUserId(userId: string): Promise<w.SavedGame[]> {
```

Comment:
You should add a `timestamp` field to the `SavedGame` interface.

Reply:
Done